spec/nameable_spec.rb

```diff
@@ -0,0 +1,10 @@
+require_relative './../decorators/nameable'
+
+describe 'Test capitalize_decorator object' do
+  nameable = Nameable.new
+  it 'takes parameters and returns a corrected name' do
+    expect do
+      nameable.correct_name
+    end.to raise_error(NotImplementedError)
+  end
```

Comment:
- [OPTIONAL] Great job so far 👏🏻 👏🏻 However, according to the project requirements, there should be unit tests for all the methods in all the classes, Kindly make sure the `add_rental` class book method has (at least) an 'it' block 👇🏻 check your book.rb file

```ruby
  def add_rental(rental)
    @rentals << rental
    rental.book = self
  end
```